system/ee/ExpressionEngine/Library/String/Str.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace ExpressionEngine\Library\String;
+
+class Str
+{
+    /**
+     * @param string $value
+     * @return string
+     */
+    public static function studly(string $value): string
+    {
+        return str_replace(' ', '',
+            ucwords(str_replace(['-', '_'], ' ', $value))
+        );
+    }
+
+    /**
+     * Converts/A/String/Like/This into a String\Formatted\Like\This
+     * @param $value
+     * @return string
+     */
+    public static function path2ns(string $value): string
+    {
+        return str_replace('/', '\\', $value);
+    }
+
+    /**
+     * Converts a string formatted-with-dashes to a CamelCaseStringInstead
+     * @param string $value
+     * @return string
+     */
+    public static function dash2ns(string $value): string
+    {
+        return self::path2ns(
+            str_replace(' ', '/',
+                str_replace('cartthrob', 'CartThrob', // ;)
+                    ucwords(str_replace('_', ' ', $value))
+                )
```

Comment:
I think hardcoded `CartThob` probably went in here by mistake?

Reply:
Yeah, can't provide special consideration to Add-ons; just won't scale lol

Will clean up.